test-projects/use-devDependency/worker.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2019 Adobe Inc. All rights reserved.
```

Comment:
Nitpick: 2020.

Reply:
This is just a copy and the original code was written in 2019 so this is correct in my view.